extensions/th2setarea.inx

```diff
@@ -14,8 +14,14 @@
     <item>snow</item>
     <item>ice</item>
     <item>pebbles</item>
+    <item>bedrock</item>
     <item>flowstone</item>
     <item>moonmilk</item>
+    <item>pillar</item>
+    <item>pillar_with_curtains</item>
```

Comment:
Does this work? _thbook-v6.2.0.pdf_ uses dashes instead of underscores (`pillar-with-curtains`)

Reply:
I created one today.